src/main/java/de/retest/recheck/cli/utils/Version.java

```diff
@@ -0,0 +1,32 @@
+package de.retest.recheck.cli.utils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class Version {
```

Comment:
I would either suggest to load the properties file only once, for example:

```java
private static final Properties props = loadProperties();

public static final String RECHECK_CLI_VERSION = props.getProperty( ... );
public static final String RECHECK_VERSION = props.getProperty( ... );
```

Or use Owner:

```java
@Sources( "classpath:retest-version.properties" )
public interface Versions extends Config {

    String recheckCliVersion();

    String recheckVersion();

}
```

```properties
recheckCliVersion=${project.version}
recheckVersion=${recheck.version}
```